src/mmw/js/src/app.js

```diff
@@ -3,6 +3,8 @@
 var $ = require('jquery'),
     _ = require('lodash'),
     Marionette = require('../shim/backbone.marionette'),
+    settings = require('./core/settings').getSettings(),
+    itsi = require('./core/itsiEmbedInterface'),
```

Comment:
We don't seem to have a convention about file naming in general. The closest thing I can find is "NTFS file/folder names are not [case sensitive], so be careful..." from out C# guidelines. I'm generally in favor of lowercase names with dashes or underscores. Traditionally I've preferred underscores.